plugins/tags/plugin.cpp

```diff
@@ -614,11 +620,11 @@ namespace golos { namespace plugins { namespace tags {
 
             for (; itr != idx.end() && itr->author == *query.start_author && result.size() < query.limit; ++itr) {
                 if (itr->parent_author.size() > 0) {
-                    discussion p(db.get<comment_object>(itr->root_comment), db);
+                    discussion p( pimpl->create_comment_api_object(db.get<comment_object>(itr->root_comment) ) );
```

Comment:
To many copies... Add an additional  to create discussion from comment_object.

Reply:
Fixed